### ISSUE TITLE
Upgrade to babel v6.26.3?

### ISSUE THREAD

**Issue by pedrofurtado** [babel/babel-standalone#102]
@Daniel15

**Comment by Daniel15** [comment]
This is maintained as part of Babel now. @hzoo Do you know if we're still updating babel-standalone for v6.x, or are we just focused on v7.x? Is it okay to push a new 6.x babel-standalone release?

**Comment by hzoo** [comment]
If you are cool with doing releases on v6 I definitely wouldn't be opposed?

**Comment by pedrofurtado** [comment]
@Daniel15 @hzoo It's a great idea to update the 6.x versions here. Many applications use this repository in production, and could be obsolete (without new features or bug fixes) while not update the version of babel standalone.

**Comment by hzoo** [comment]
I don't think we ever intended Babel standalone to be used in "prod" but interesting to hear that. It's also not actively maintained if that wasn't clear since we have been trying to release 7. We definitely don't need to add features to 6.x or even update it much if at all which is why the last updates were only for sourcemaps which isn't really useful for Babel standalone anyway

**Comment by pedrofurtado** [comment]
@hzoo @Daniel15  An example is this repository: https://github.com/babel/ruby-babel-transpiler . It uses the babel standalone source code, and many Rails applications that I work on use it. But, unfortunately, it is not updated with 6.x versions of Babel.

**Comment by pedrofurtado** [comment]
@hzoo @Daniel15 There are some issues related to this: 

https://github.com/babel/ruby-babel-transpiler/issues/288
https://github.com/babel/ruby-babel-transpiler/issues/300

Upgrade to 6.x version is a great improvement for all community that uses the babel standalone, like `ruby-babel-transpiler` in Rails applications :tada:

**Comment by Daniel15** [comment]
I'll push an update later this week, maybe tomorrow. Hopefully the update script still works, otherwise I'll just do it manually.

These releases are actually supposed to be automated, but the automation broke at some point (eg. see https://build.dan.cx/job/babel-standalone-update/31/console) 😢

**Comment by Daniel15** [comment]
@pedrofurtado Both of those issues are specific to ruby-babel-transpiler, and most of the work for those would need to be done in that project. Once ruby-babel-transpiler is updated to use babel-standalone, switching to a different version should be easy.

**Comment by pedrofurtado** [comment]
@Daniel15 Thanks for feedback! Any news about updating of version, let me know that I will help you with tests (locally on my machine and in the applications that I work here). :+1:

**Comment by pedrofurtado** [comment]
@Daniel15 @hzoo Any news?
